tests/eosd_run_test.py

```diff
@@ -36,6 +37,7 @@ def cmdError(name, code=0, exitNow=False):
 parser.add_argument("-o", "--output", type=str, help="output file", default=TEST_OUTPUT_DEFAULT)
 parser.add_argument("-h", "--host", type=str, help="eosd host name", default=LOCAL_HOST)
 parser.add_argument("-p", "--port", type=int, help="eosd host port", default=DEFAULT_PORT)
+parser.add_argument("--mongoDb", help="Configure a MongoDb instance", action='store_true')
 parser.add_argument("--dumpErrorDetails",
                     help="Upon error print tn_data_*/config.ini and tn_data_*/stderr.log to stdout",
```

Comment:
In general, the team does not like CamelCase. When moving over to eos-noon lets change all of these to `--mongodb` and `--dump-error-details`, etc.